_assets/stylesheets/components/breadcrumbs.scss

```diff
@@ -36,13 +36,12 @@
 }
 
 .acc-breadcrumb-home {
-  background-image: image_url('icons/globe.svg');
   background-position: center left;
   background-repeat: no-repeat;
   background-size: 1.6em;
   padding: 0;
   text-indent: -9999px;
-  width: 2.4em;
+  cursor: default;
```

Comment:
I don't understand; do we not want the globe there at all? We should just remove it if that's the case.